templates/kubernetes/terraform/environments/prod/main.tf

```diff
@@ -42,3 +42,11 @@ module "kubernetes" {
     }
     # could be more policies defined here (if have)
   ]
+
+  # Wireguard configuration
+  vpn_server_address = "10.10.199.0/24"
+  vpn_client_publickeys = [
+    { "10.10.199.201/32" = "/B3Q/Hlf+ILInjpehTLk9DZGgybdGdbm0SsG87OnWV0=" }, # client IP1
```

Comment:
How about a `list(tuple([string, string, string]))` here like : `[ { "person name", "ip", "pubkey" } ]` ? Then you could template the person's name in as a comment for each peer block.

Reply:
done